### PR TITLE
init rspec

### PR DESCRIPTION
**Pull request by haupt-2855**
## Related Tickets
- [#45392](https://edu-redmine.sun-asterisk.vn/issues/45392)

## WHAT (optional)
- Init rspec
- Add gem rspec-rails
- Add gem do coverage
- Setup CI local for rspec

## Evidence (Scre
Screenshot from 2022-01-19 17-28-34
enshot or Video)

### CODE REVIEW

**Comment by haupt-2855** [comment]
ready

File: spec/helpers/admin/products_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```
Comment (vanvtt-0952) [inline spec/helpers/admin/products_helper_spec.rb]: mấy spec chưa viết thì xóa đi em

**Comment by haupt-2855** [comment]
ready

**Comment by vanvtt-0952** [comment]
commented
ticket đâu bé

**Comment by haupt-2855** [comment]
ready

**Pull request merged**